packages/react-instantsearch-core/src/core/indexUtils.js

```diff
@@ -6,6 +6,21 @@ export function getIndexId(context) {
     : context.ais.mainTargetedIndex;
 }
 
+export function addAbsolutePositions(hits, hitsPerPage, page) {
```

Comment:
It should be more inside `utils` rather than `indexUtils`. This file mainly deals with `context` and `searchState`.

Reply:
Thanks for advising on that, I initially has a FIXME: find theses functions a better place but linting got to remove it :)
Will do